From eba65779cd95716706912707db297d389fecf8e5 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Sun, 2 Aug 2009 19:49:27 +0200
Subject: [PATCH 19/19] add keymap for a HP Pavillion dv6315ea

Added keymap for a HP Pavillion dv6315ea with empty
system.hardware.* keys.
---
 fdi/information/10freedesktop/30-keymap-hp.fdi |    9 +++++++++
 1 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/fdi/information/10freedesktop/30-keymap-hp.fdi b/fdi/information/10freedesktop/30-keymap-hp.fdi
index 8df6185..3bc4eb1 100644
--- a/fdi/information/10freedesktop/30-keymap-hp.fdi
+++ b/fdi/information/10freedesktop/30-keymap-hp.fdi
@@ -56,6 +56,15 @@
             <append key="input.keymap.data" type="strlist">e059:f23</append>    <!-- touchpad on  -->
 	  </match>
 
+	  <match key="/org/freedesktop/Hal/devices/computer:system.board.vendor" string="Quanta">
+	    <!-- HP Pavillion dv6315ea with empty system.hardware.* keys fd.o#19936 -->
+	    <match key="/org/freedesktop/Hal/devices/computer:system.board.product" string="30B7">
+	      <match key="/org/freedesktop/Hal/devices/computer:system.board.version" string="65.2B">
+                <append key="input.keymap.data" type="strlist">e008:media</append> <!-- FIXME: quick play -->
+	      </match>
+	     </match>
+	  </match>
+
       	</match>
       </match>
     </match>
-- 
1.6.4.4

